src/elements/tabs/tab-group/tab-group.stories.ts

```diff
@@ -34,38 +45,47 @@ const tabPanelOne = (): TemplateResult => html`
     Diam maecenas ultricies mi eget mauris pharetra et ultrices neque ornare aenean euismod
     elementum nisi quis eleifend quam adipiscing vitae proin sagittis nisl rhoncus mattis rhoncus
     urna neque viverra justo nec.
-  </div>
+  </sbb-tab>
 `;
 
 const tabPanelTwo = (): TemplateResult => html`
-  <section>
+  <sbb-tab>
     Diam maecenas ultricies mi eget mauris pharetra et ultrices neque ornare aenean euismod
     elementum nisi quis eleifend quam adipiscing vitae proin sagittis nisl rhoncus mattis rhoncus
     urna neque viverra justo nec.
     <sbb-block-link target="_blank" href="https://www.sbb.ch">Visit sbb.ch</sbb-block-link>
-  </section>
+  </sbb-tab>
 `;
 
 const tabPanelFour = (): TemplateResult => html`
-  <article>
+  <sbb-tab>
```

Comment:
I think inside the tab it would be great to have a p tag or article or whatever so it has a clear role. (would have to be corrected in all stories and readme and examples)